PlayCover/Views/MenuBarView.swift

```diff
@@ -109,7 +109,7 @@ struct PlayCoverViewMenuView: Commands {
         CommandGroup(before: .sidebar) {
             Button("menubar.clearCache") {
                 DataCache.instance.cleanAll()
-                URLCache.iconCache.removeAllCachedResponses()
+                Cacher().removeImageCache()
```

Comment:
A shared instance of `Cacher` should be called instead of an initialized instance.